javascript/atoms/dom.js

```diff
@@ -312,18 +313,18 @@ bot.dom.getAttribute = function(element, attributeName) {
 /**
  * List of elements that support the "disabled" attribute, as defined by the
  * HTML 4.01 specification.
- * @private {!Array.<goog.dom.TagName>}
+ * @private {!Object<string, boolean>}
  * @const
  * @see http://www.w3.org/TR/html401/interact/forms.html#h-17.12.1
  */
-bot.dom.DISABLED_ATTRIBUTE_SUPPORTED_ = [
+bot.dom.DISABLED_ATTRIBUTE_SUPPORTED_ = goog.object.createSet([
   goog.dom.TagName.BUTTON,
```

Comment:
Shouldn't these all be wrapped in String(goog.dom.TagName...) then?

Assuming they are adding a toString function to them?